README.md

```diff
@@ -153,7 +153,75 @@ Example output:
 
 # Usage in CI Systems
 
-> 🚧 COMING SOON 🚧
+> 🚧 COMING SOON: native docker image & GitHub action 🚧
+
+## GitHub Actions workflows
+
+Here is an example of a simaple GitHub Actions workflow that uses `kubectl-validate` to validate Kubernetes manifests.
+This workflow will run on every pull request to the `main` branch and will fail if any of the manifests in the dir `k8s-manifest/` are invalid.
+
+```yaml
+name: kubectl-validate
+on:
+  pull_request:
+    branches:
+      - main
+  
+env:
+  MANIFESTS_PATH: 'k8s-manifest/*.yaml'
```

Comment:
Also I don't think you should have to use `*.yaml` in general, the program only picks json/yaml by default.

Reply:
I didn't know that and apparently, it's also working recursively by default. 
it's worth mentioning these two in the docs 😅

Reply:
Sounds like you just got yourself a new task ;-)

Reply:
😂 
will happily help with that but first we will need to merge this PR to avoid conflicts